figures/what_you_can_see_with_lfp/npx/Fig3-figure_traub_timespace.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+@author: mbejtka
+"""
+import numpy as np
+import h5py as h5
+import matplotlib.pyplot as plt
+from Fig2_traub_data_kcsd_column_figure import (prepare_electrodes, prepare_pots, do_kcsd)
+import kCSD2D_reconstruction_from_npx as npx
+from scipy.signal import filtfilt, butter
+
+
+def set_axis(ax, letter=None):
+    ax.text(
+        -0.05,
+        1.05,
+        letter,
+        fontsize=20,
+        weight='bold',
+        transform=ax.transAxes)
```

Comment:
Call formatting inconsistent with used in the project. Is there a reason for that?